pkg/lib/config/identity.go

```diff
@@ -352,6 +352,29 @@ func (c OAuthSSOProviderConfig) IsMissingCredentialAllowed() bool {
 	return v
 }
 
+type OAuthProviderStatus string
+
+const (
+	OAuthProviderStatusActive               OAuthProviderStatus = "active"
+	OAuthProviderStatusMissingCredentials   OAuthProviderStatus = "missing_credentials"
+	OAuthProviderStatusUsingDemoCredentials OAuthProviderStatus = "using_demo_credentials" // nolint: gosec
```

Comment:
Rename to `using_shared_credentials`

Reply:
Similar to the secret key, I prefer name it demo credential.
In auth ui, we also call this as demo credential.